admins/pageflow/entry.rb

```diff
@@ -157,6 +187,17 @@ def scoped_collection
         params.key?(:folder_id) ? result.where(folder_id: params[:folder_id]) : result
       end
 
+      def build_new_resource
+        super.tap do |entry|
+          entry.account ||=
+            AccountPolicy::Scope.new(current_user,
+                                               Pageflow::Account).entry_creatable.first ||
+            Pageflow::Account.first
```

Comment:
Many indentations fell prey to mass removing of Pageflow::Policies:: and Policies::